packages/expo-video/CHANGELOG.md

```diff
@@ -4,12 +4,17 @@
 
 ### 🛠 Breaking changes
 
+- [Android] In order to show the now playing notification, the `supportsBackgroundPlayground` property of the config plugin has to be `true`. ([#38980](https://github.com/expo/expo/pull/38980) by [@kerwanp](https://github.com/kerwanp))
```

Comment:
seems like a lot of changelog for a single PR – maybe just keep this in a single category?